### DISCUSSION TITLE
Black and White PCB issues at PCBway

### DISCUSSION THREAD

**Discussion by jonnypoko** [frntc/SIDKick-pico#75] [Q&A]
Clicked on the add to cart for the RP2040 boards and the audit came back from PCBway with "1. the distance between ic is less than 0.22mm (attached pic is one part) For the black&matte black&white solder mask, only the ic distance more than 0.22mm that we can make the bridge(add solder ink between pins);  The solder bridge can prevent the short circuit. We suggest to update solder mask color. Pls note this is not the problem in design. Because it is ok for green, red, blue solder mask. Pls check and advise." - is this an issue? (I presumed a small bridge on the top layer could be fixed) did you get this at all? and is it anything to do with the HINT on the project page? - I'm afraid I am too much of a noob to understand it as I cant see any of those options anywhere when I ordered?

**Comment by frntc** [comment]
I got this warning initially and then revised the boards before I ordered them myself. Can you refer them to my original order from PCBWay and ask them to take the settings from there? I had my test batch produced in black and it was OK (and no more complaints from them).

**Reply by jonnypoko** [reply]
I'll give it a go
